shakenfist_client/main.py

```diff
@@ -912,12 +912,18 @@ def _parse_spec(spec):
 @click.option('-p', '--placement', type=click.STRING)
 @click.option('-V', '--videospec', type=click.STRING)
 @click.option('--bios/--uefi', is_flag=True, default=True)
+@click.option('--force', is_flag=True, default=False)
 @click.option('--namespace', type=click.STRING)
 @click.pass_context
 def instance_create(ctx, name=None, cpus=None, memory=None, network=None, floated=None,
                     networkspec=None, disk=None, diskspec=None, sshkey=None, sshkeydata=None,
                     userdata=None, encodeduserdata=None, placement=None, videospec=None,
-                    namespace=None, bios=True):
+                    namespace=None, bios=True, force=False):
+    if memory < 128 and not force:
+        print(f'You have specified {memory}MB of memory. '
```

Comment:
Sorry to be nit picky, but we don't use this style of format string anywhere else. I think we should explain why we're telling them about that amount of memory. Could you do something like:

"You have specified %dmb of memory, which seems very low. "

Instead?

Reply:
Sounds fair. Standby.